pkg/pxc/app/statefulset/node.go

```diff
@@ -153,14 +160,45 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string) corev1.Container
 			},
 		},
 	}
-
-	return appc
+	if compareVersion130 < 0 {
+		appc.VolumeMounts = append(appc.VolumeMounts, corev1.VolumeMount{
+			Name:      "config",
+			MountPath: "/etc/mysql/conf.d",
+		})
+	}
```

Comment:
why do you need a condition here? my understanding is that that volume should be present unconditionally. just remove `if` here and `/etc/mysql/conf.d` volume from condition below.